src/components/app/reducers.js

```diff
@@ -1,51 +1,24 @@
-import Cookies from 'cookies-js';
-
-import { SET_COOKIE, REMOVE_COOKIE, RESET_STATE, RESET_STATUS_STATE } from './actionTypes';
-import { EXPIRATION } from './constants';
+import { GET_USER_DATA } from './actionTypes';
+import Auth from '../auth0/Auth';
+const auth = new Auth();
 
 const initialState = {
-    isLoggedIn: null,
-    status: "",
-    token: "",
+    isLoggedIn: auth.isAuthenticated(),
```

Comment:
initially `isLoggedIn: null`, only a request to authenticate should update this through the action to reducer flow. not calling a function here. This will need to bugs down the line

Reply:
ok, cool